xml_inscriptions/metadata_only/CA.Malibu.JPGM.ETR.82.AD.25.xml

```diff
@@ -0,0 +1,164 @@
+
+                        <!--  *************************************  <objectDesc>  *********************************  -->
+
+                        <objectDesc ana="#instrumentum">
+                            <supportDesc ana="#clay">
+                                <support>
+                                    <p> Terracotta bulla or seal</p>
+                                    <dimensions type="surface" unit="cm">
+                                        <height>5.8</height>
+                                        <width>4.9</width>
+                                        <depth>2.2</depth>
+                                    </dimensions>
+                                </support>
+                                <condition ana="#complete">
+                                    <p/>
+                                </condition>
+                            </supportDesc>
+                            <!--  *************************************  <layoutDesc>  ********************************* -->
+                            <layoutDesc>
+                                <layout columns="0" writtenLines="0">
+                                    <p/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+
+                        <!--  *************************************  <handDesc>  ********************************* -->
+
+                        <handDesc hands="1">
+                            <handNote ana="#impressed"> </handNote>
+                        </handDesc>
+
+                        <!--  *************************************  <decoDesc>  *********************************     -->
+                        <decoDesc>
+                            <decoNote>
+                                <p/>
+                            </decoNote>
+                        </decoDesc>
+                    </physDesc>
+
+                    <!--  *************************************  <history>  *********************************  -->
+
+                    <history>
+
+                        <summary>
+                            <rs/>  <!-- content can be "fake" or blank -->
+                        </summary>
+
+                        <origin>
+                            <date notBefore="0501" notAfter="0599">6th century CE</date>
+                            <placeName ref="asia.asia_minor">Persia</placeName>
+                        </origin>
+
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#"/>
+                        <biblScope unit="page">00</biblScope>
+                    </bibl>
+                    <bibl>
+                        <ref type="external" target="https://www.jstor.org/stable/4166525">Koch,
```

Comment:
This needs to be in the regular bibliography and linked through there -- it's not an external link based on the way that we handle this.